docs/src/pages/components/text-fields/text-fields.md

```diff
@@ -168,6 +168,35 @@ For number validation, one viable alternative is to use the default input type="
 
 In the future, we might provide a [number input component](https://github.com/mui-org/material-ui/issues/19154).
 
+
+### Helper text
+
+The helper text prop effects the height of the text field. If two text fields, one with a helper text one without, are placed side by side they will be at different heights. For example the following code: 
+
+```jsx
+<div style={{ display: 'flex', alignItems: 'center' }}>
+	<TextField helperText="Please enter your name" />
+	<TextField />
+</div>
+```
+
+Will result in this:
+
+![helper-text-1](/static/images/text-fields/helper-text-1.png)
```

Comment:
Should we have a live example here?

Reply:
Ok sure I can change the images to live versions